doozerlib/cli/images_streams.py

```diff
@@ -530,6 +532,35 @@ def prs():
     pass
 
 
+@prs.command('list', short_help='List all open prs for upstream repo, run this need GITHUB_TOKEN var from env got set.')
```

Comment:
I would reword a little bit shorter:
```suggestion
@prs.command('list', short_help='List all open prs for upstream repos (requires GITHUB_TOKEN env var to be set.')
```